payloads/v8/invite.ts

```diff
@@ -68,6 +69,10 @@ export interface APIInvite {
 	 * The stage instance data if there is a public stage instance in the stage channel this invite is for
 	 */
 	stage_instance?: APIInviteStageInstance;
+	/**
+	 * The guild scheduled event data, only included if `guild_scheduled_event_id` contains a valid guild scheduled event id
```

Comment:
If this is a query parameter, can you follow the same flow as the others? (like expires_at for example)